pkg/volume/glusterfs/glusterfs_util.go

```diff
@@ -0,0 +1,58 @@
+package glusterfs
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+
```

Comment:
this is actually idiomatic for go -- the idiom is to separate stdlib and third-party imports with a newline